pipeline.py

```diff
@@ -51,19 +60,26 @@ class CorpusManager:
     """
 
     def __init__(self, path_to_raw_txt_data: str):
-        pass
+        self.path_to_raw_txt_data = path_to_raw_txt_data
+        self._storage = {}
+        self._scan_dataset()
 
     def _scan_dataset(self):
         """
         Register each dataset entry
         """
-        pass
+        path_to_raw_txt_data = Path(self.path_to_raw_txt_data)
+
+        for file in path_to_raw_txt_data.glob('*'):
+            if '_raw.txt' in str(file):
```

Comment:
consider using `file.name`, it is designed especially for such cases and is more readable